pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -287,6 +287,32 @@ func createNodeCRD(clientset apiextensionsclient.Interface) error {
 	return createUpdateCRD(clientset, "v2.CiliumNode", res)
 }
 
+// createIdentityCRD creates and updates the CiliumIdentity CRD. It should be
+// called on agent startup but is idempotent and safe to call again.
+func createIdentityCRD(clientset apiextensionsclient.Interface) error {
+	res := &apiextensionsv1beta1.CustomResourceDefinition{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "ciliumidentities." + SchemeGroupVersion.Group,
```

Comment:
Perhaps hardcode `ciliumidentities` and `ciliumidentity` to consts in case they are used elsewhere?

Reply:
ok, but 3/4 CRDs have the strings used inline. I switched it to be like the CEP creator. There are no global constants specific to the CRDs in this file.